yazi fails to build for Android

### What system are you running Yazi on?

Linux X11

### What terminal are you running Yazi in?

termux

### `yazi --debug` output

N/A

### Did you try the latest nightly build to see if the problem got fixed?

No, and I'll explain why below

### Describe the bug

compiler error.

```
Compiling yazi-scheduler v0.3.0 (/home/builder/.termux-build/yazi/src/yazi-scheduler)
error[E0433]: failed to resolve: use of undeclared crate or module trash
--> yazi-scheduler/src/file/file.rs:143:14
|
143 |                     Ok::<_, trash::Error>(())
|                             ^^^^^ use of undeclared crate or module trash
For more information about this error, try rustc --explain E0433.
```


### Expected Behavior

The project should compile without any error.

### To Reproduce

Try to compile yazi for aarch64-linux-android target.

### Configuration

_No response_

### Anything else?

_No response_

--- a/yazi-scheduler/src/file/file.rs
+++ b/yazi-scheduler/src/file/file.rs
@@ -140,7 +140,7 @@ impl File {
 					{
 						trash::delete(&task.target)?;
 					}
-					Ok::<_, trash::Error>(())
+					Ok::<_, anyhow::Error>(())
 				})
 				.await??;
 				self.prog.send(TaskProg::Adv(task.id, 1, task.length))?;

